### ISSUE TITLE
AMD drivers not recognized

### ISSUE THREAD

**Issue by nono71x** [develsoftware/GMinerRelease#919]
Gminer does not work, no algo works on the 6000 series. I have 6800 6800xt 6700xt and nothing works, I am willing that on the latest drivers it could be a problem, but you have to go back 5 years to only have a few algos . We need to stop betting everything on Nvidia all the time, and maybe get back to the basics of AMD.

**Comment by Ph0enix1127** [comment]
True, on my nvidia cards everything works so perfectly and all algos are supported, AMD has so much algos that don't work and just issues and bugs all the time..
